service/router.go

```diff
@@ -30,6 +30,7 @@ func setupRouter(h handler) *mux.Router {
 	r.HandleFunc("/projects/{projectName}/targets/{targetName}", h.updateTarget).Methods(http.MethodPatch)
 	r.HandleFunc("/projects/{projectName}/targets/{targetName}/operations", h.createWorkflowFromGit).Methods(http.MethodPost)
 	r.HandleFunc("/projects/{projectName}/targets/{targetName}/workflows", h.listWorkflows).Methods(http.MethodGet)
+	r.HandleFunc("/projects/{projectName}/tokens", h.createToken).Methods(http.MethodPost)
```

Comment:
@Acepie reminded me in my PR, but can we add this to https://github.com/cello-proj/cello/blob/main/docs/developers/api.md?